python/samples/concepts/memory/data_models.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft. All rights reserved.
+from dataclasses import dataclass, field
+from typing import Annotated, Any
+from uuid import UUID, uuid4
+
+from pydantic import Field
+
+from semantic_kernel.data.vector_store_model_decorator import vectorstoremodel
+from semantic_kernel.data.vector_store_record_fields import (
+    VectorStoreRecordDataField,
+    VectorStoreRecordKeyField,
+    VectorStoreRecordVectorField,
+)
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
```

Comment:
I think it would be helpful for samples, concepts, etc. to have comments describing the concept that we're trying to show. In this case something simple about how you can define vector store models in several different ways depending on your needs.

Reply:
Same comment for all the others.